after/ftplugin/xml.vim

```diff
@@ -1,12 +1,6 @@
 if gradle#isGradleProject()
-
-  call gradle#setClassPath()
```

Comment:
Just realized that the gradle vim tasks totally fails if the AndroidManifest.xml has syntax errors. This makes it impossible to detect if the project is an android one. I need to rethink again the best way to determine if a project is an Android project.